packages/backend-common/config.d.ts

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export interface Config {
+  app: {
+    baseUrl: string; // defined in core, but repeated here without doc
```

Comment:
maybe we can standardise on `/** @see @backstage/core */`, but maybe we don't want jsdocs in these cases?

Reply:
Yeah in general this doesn't fully establish a pattern for how to deal with values that are defined in a different package but re-used. We do indeed want to avoid jsdoc here though, as that could potentially override the original docs, and something like `@see @backstage/core` would be merged in with the actual definition which would be confusing.